test/decompile/basic.txt

```diff
@@ -41,6 +41,9 @@
     end
     i32.const 1
   )
+  ;; LLD outputs a name section with de-mangled C++ function signatures as names,
+  ;; so have to make sure special chars get removed.
+  (func (export "signature-&<>()") (param) (result))
```

Comment:
I'd add another test case of like `export "signature[]{}+/"` or something that would collide with the rewritten `signature______` identifier to check deduping. And if that doesn't work yet leave a note about it

Reply:
We're not deduping yet. Let me add a FIXME

Reply:
actually we are, doh.